File: 70P/Find-All-Missing-Numbers.py
nums1 = [4,3,2,7,8,2,3,1]

nums2 = [1,1]


def find_all(numbs):
    setn = set(numbs)
    actual_range = range(1, len(numbs))
    ans = []
    for i in actual_range:
        if i not in setn:
            ans.append(i)
            
    return ans


ans = find_all(nums1)
print(ans)

File: 70P/Find-All-Missing-Numbers.js
//Input: nums = [4,3,2,7,8,2,3,1]
//Output: [5,6]

let nums1 = [4,3,2,7,8,2,3,1]

let nums2 = [1,1]

function findAll(numbs){
    const setN = new Set(numbs);
    const n = numbs.length;
    let ans = [];

    for (let i = 1; i <= n; i++){
        if(!setN.has(i)){
            ans.push(i);
        }
    }

    return ans
}

let theAns = findAll(nums2);

console.log(theAns);